### PR TITLE
Units Documentation

### LINKED ISSUE #491: Add documentation for new units functionality
All the new units work will need quite a bit of documentation. I think this breaks down into:

- [ ] A "What's new" entry
- [ ] Some longer discussion on the "Data import" page of exactly what the data returned is by the import methods, including links to the relevant Sunpy/Astropy/Pandas documentation (http://docs.heliopy.org/en/latest/data/index.html)
- [ ] Check that all the converted functions have the correct docstrings for the returns section

### PR DESCRIPTION
**Pull request by abotiamnot**
Fixes #491 

- [x] What's New?

- [x] Entry on Data Import Page

- [x] Docstrings ( of the modules that have been converted )

- [x] Add an example to the data import page

### CODE REVIEW

**Comment by abotiamnot** [comment]
Docstrings need to be added later on since there's still a few modules that don't use units

File: CHANGES.rst [hunk 1]
```diff
@@ -3,13 +3,22 @@ Version 0.6.0
...
+  HelioPy modules now return units.
+- Added a new :func:`.data.util.cdf_units` function that can extract UNIT attribute from
+  the CDF files.
 
 Backwards incompatible changes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - :func:`.data.helios.merged`, :func:`.data.helios.mag_4hz`,
   :func:`.data.helios.corefit` and :func:`.data.helios.mag_ness` no longer take
   a `verbose` keyword argument. :issue:`467`
+- HelioPy no longer returns a Pandas DataFrame.
```
Comment (dstansby) [inline CHANGES.rst]: Can this be something like "The methods in :mod:`heliopy.data` no longer return...".

(BTW if you want to learn more about sphinx markup then http://www.sphinx-doc.org/en/stable/markup/inline.html is a useful page)

File: CHANGES.rst [hunk 2]
```diff
@@ -3,13 +3,22 @@ Version 0.6.0
...
 
+Fixed bugs
+^^^^^^^^^^
+- :func:`.data.mms.fpi_dis_moms` has been updated to the V3.3.0.
```
Comment (dstansby) [inline CHANGES.rst]: I put this fix in the recently released v0.5.2, so no need to put a chnagelog entry here.

File: doc/source/data/index.rst [hunk 3]
```diff
@@ -45,3 +45,17 @@ util module:
    :maxdepth: 1
 
    util
+
+SunPy and AstroPy Integration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+HelioPy is built to be used alongside SunPy and AstroPy. Before V0.6, HelioPy returned
```
Comment (dstansby) [inline doc/source/data/index.rst]: - Can SunPy and AstroPy get links to their home pages
- V0.6 -> v0.6 (I think it looks better small case)

File: doc/source/data/index.rst [hunk 4]
```diff
@@ -45,3 +45,17 @@ util module:
...
+a Pandas DataFrame object. After implementation of physical units into the data, HelioPy
+returns a SunPy TimeSeries object. The TimeSeries object is capable of storing data like Pandas DataFrame,
+with added benefit of being used alongside AstroPy units.
+
+SunPy TimeSeries Documentation is available
+at http://docs.sunpy.org/en/v0.9.0/guide/data_types/timeseries.html
+
+AstroPy Units Documentation is available
+at http://docs.astropy.org/en/stable/units/
```
Comment (dstansby) [inline doc/source/data/index.rst]: Can you add a link to the HelioPy example that uses units?

File: heliopy/data/ace.py [hunk 5]
```diff
@@ -81,7 +81,7 @@ def mfi_h0(starttime, endtime):
 
     Returns
     -------
-    data : DataFrame
+    data : sunpy.timeseries.timeseriesbase.GenericTimeSeries
```
Comment (dstansby) [inline heliopy/data/ace.py]: These aren't showing up as links in the docs, they might need a :class: bit and ` around them.

File: heliopy/data/artemis.py [hunk 6]
```diff
@@ -8,6 +8,7 @@
 
 import pandas as pd
 import numpy as np
+import sunpy
```
Comment (dstansby) [inline heliopy/data/artemis.py]: This import isn't needed.

File: CHANGES.rst (hunk 1 above)
Comment (abotiamnot) [reply CHANGES.rst]: Thanks for the link! Sphinx is pretty new to me, but I find it an extremely great way to document stuff!

File: CHANGES.rst [hunk 7]
```diff
@@ -3,13 +3,21 @@ Version 0.6.0
 
 New features
 ^^^^^^^^^^^^
+- HelioPy has been integrated with SunPy TimeSeries and AstroPy Units. All of
+  HelioPy modules now return units.
```
Comment (dstansby) [inline CHANGES.rst]: - "All of HelioPy modules: > "All of the HelioPy data modules"

File: CHANGES.rst [hunk 8]
```diff
@@ -3,13 +3,21 @@ Version 0.6.0
...
+- Added a new :func:`.data.util.cdf_units` function that can extract UNIT attribute from
```
Comment (dstansby) [inline CHANGES.rst]: - "extract UNIT" > "extract the UNIT"

File: CHANGES.rst [hunk 9]
```diff
@@ -3,13 +3,21 @@ Version 0.6.0
...
+  the CDF files.
 
 Backwards incompatible changes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - :func:`.data.helios.merged`, :func:`.data.helios.mag_4hz`,
   :func:`.data.helios.corefit` and :func:`.data.helios.mag_ness` no longer take
   a `verbose` keyword argument. :issue:`467`
+- :mod:`heliopy.data` no longer returns a Pandas DataFrame.
```
Comment (dstansby) [inline CHANGES.rst]: - ":mod:`heliopy.data` no longer" > "Methods in :mod:`heliopy.data` no longer"

File: doc/source/data/index.rst [hunk 10]
```diff
@@ -45,3 +45,19 @@ util module:
...
+HelioPy is built to be used alongside SunPy and AstroPy. Before v0.6, HelioPy returned
+a Pandas DataFrame object. After implementation of physical units into the data, HelioPy
+returns a SunPy TimeSeries object. The TimeSeries object is capable of storing data like Pandas DataFrame,
+with added benefit of being used alongside AstroPy units.
+
+SunPy Documentation is available at http://docs.sunpy.org/
+You can also read more about `TimeSeries <http://docs.sunpy.org/en/stable/guide/data_types/timeseries.html>`_
+
+AstroPy Documentation is available at http://docs.astropy.org/
+You can also read more about `TimeSeries <http://docs.astropy.org/en/stable/units/>`_
```
Comment (dstansby) [inline doc/source/data/index.rst]: This should be units, not TimeSeries!

File: doc/source/data/index.rst [hunk 11]
```diff
@@ -45,3 +45,19 @@ util module:
...
+
+An example on how to use TimeSeries Data and AstroPy Units is also available in the Examples section.
```
Comment (dstansby) [inline doc/source/data/index.rst]: Could you add a link to the examples section? I'm not sure how to, but the spinx-gallery docs should help: https://sphinx-gallery.github.io/

Comment (abotiamnot) [reply doc/source/data/index.rst]: Got it! Thanks a lot for the reference!

**Comment by abotiamnot** [comment]
Still need to add the link to the example on data import page

**Comment by abotiamnot** [comment]
Yay! Finally! @dstansby Can you take a look now?

**Pull request merged**